onos-config/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: onos-config
-version: 1.6.4
+version: 1.6.5
```

Comment:
perhaps you need to rebase. i guess a PR merged in the morning

Reply:
ah that was exactly what I was hoping to avoid :) 

thanks for the catch